src/test/resources/com/puppycrawl/tools/checkstyle/checks/imports/unusedimports/InputUnusedImports.java

```diff
@@ -5,7 +5,7 @@
 package com.puppycrawl.tools.checkstyle.checks.imports.unusedimports;
 
 import com.puppycrawl.tools.checkstyle.checks.imports.*;
-                                     import com.puppycrawl.tools.checkstyle.checks.imports.InputImportBug;
+import com.puppycrawl.tools.checkstyle.checks.imports.unusedimports.InputUnusedImportsBug;
```

Comment:
please restore weird indentation, this file is for testing, the more weird file the better. Just change package and name of class.